docs/docsource/installation/install/install_xdm.rst

```diff
@@ -238,13 +235,11 @@ Deploy |product|
         version: |version|
         replicas: 1
         mongodb:
-          provider: KubeDB
           persistence:
             volumeClaimTemplate:
               size: 300Gi
               storageClassName: sc-300-g
         redis:
-          provider: KubeDB
```

Comment:
same for redis, it is not about removing the provider: a provider still needs to be specified, and parameter adjusted accordingly.